### PR TITLE
Set Up Main Page

### PR DESCRIPTION
**Pull request by BruceJi7**

### CODE REVIEW

**Comment by BruceJi7** [comment]
I will add some more updated icons, stand by

File: src/pages/Home/MainSection/ActionsSection/index.tsx [hunk 1]
```diff
@@ -1,21 +1,42 @@
 import Link from "next/link";
 
+import BuyIcon from "@/assets/icons/buy-icon.svg";
+import SwapIcon from "@/assets/icons/swap-icon.svg";
+import SendIcon from "@/assets/icons/send-icon.svg";
+
 import styles from "./ActionsSection.module.scss";
+import TransactionHistory from "./TransactionHistory";
```
Comment (ivanms1) [inline src/pages/Home/MainSection/ActionsSection/index.tsx:8]: order

File: src/pages/Home/MainSection/TokenCard/index.tsx [hunk 2]
```diff
@@ -42,30 +42,23 @@ function TokenCard({ token, i }: TokenCardProps) {
...
+          <span className={styles.TokenLabel}>{token.token}</span>
         </div>
-        <div className={styles.WalletAddressContainer}>
-          <div className={styles.WalletAddress}>
-            {currentUser?.walletAddress}
+        <div className={styles.Details}>
+          <div className={styles.LabelLine}>
+            <span className={styles.Label}>WalletID</span>
+
+            <CopyIcon />
```
Comment (ivanms1) [inline src/pages/Home/MainSection/TokenCard/index.tsx:58]: could add functionality to this?

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: src/pages/Home/MainSection/TokenCard/index.tsx (hunk 2 above)
Comment (BruceJi7) [reply src/pages/Home/MainSection/TokenCard/index.tsx:58]: To the copy icon? It wil become a button later

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**